lib/api/cli/index.js

```diff
@@ -1,91 +1,109 @@
+      adminExists: new Action(),
+      clearCache: new Action(),
+      cleanDb: new Action(),
+      createFirstAdmin: new Action(),
+      data: new Action(),
+      dump: new Action(),
+      managePlugins: new Action({
+        timeout: 1000,
+        timeOutCB: () => {
+          if (!this.maxTimeout) {
+            this.maxTimeout = 5 * 60 * 1000;
+            this.spent = this.timeout;
+          }
+
+          this.spent += this.timeout;
+
+          if (this.spent < this.maxTimeout) {
+            process.stdout.write('.');
+            this.initTimeout();
+          }
+          else {
+            console.error(`ERROR: No response from Kuzzle within ̀${this.maxTimeout / 1000}s. Exiting`); // eslint-disable-line no-console
+            process.exit(1);
+          }
         }
+      })
+    };
+  }
 
-        this.spent += this.timeout;
-
-        if (this.spent < this.maxTimeout) {
-          process.stdout.write('.');
-          return this.initTimeout();
-        }
-
-        console.error(`ERROR: No response from Kuzzle within ̀${this.maxTimeout / 1000}s. Exiting`); // eslint-disable-line no-console
-        process.exit(1);
-      }
-    })
-  };
-
-  this.do = function cliDo (actionIdentifier, data, params) {
-    let
-      action = this.actions[actionIdentifier];
+  doAction(actionIdentifier, data, params) {
+    const action = this.actions[actionIdentifier];
 
-    kuzzle.config = require('../../config');
+    this.kuzzle.config = require('../../config');
 
-    return kuzzle.internalEngine.init()
+    return this.kuzzle.internalEngine.init()
       .then(() => {
         action.initTimeout();
 
         /** @type {Service[]} */
-        kuzzle.services.list = {
+        this.kuzzle.services.list = {
           broker: new InternalBroker(kuzzle, {client: true}, kuzzle.config.services.internalBroker)
         };
 
-        return kuzzle.services.list.broker.init();
+        return this.kuzzle.services.list.broker.init();
       })
       .then(() => {
-        let
-          onListenCB = action.onListenCB.bind(action),
+        const
           raw = action.prepareData(data),
-          beforeEvent = kuzzle.funnel.getEventName('cli', 'before', actionIdentifier),
-          request;
+          beforeEvent = this.kuzzle.funnel.getEventName('cli', 'before', actionIdentifier);
 
         Object.assign(raw, {controller: 'actions', action: actionIdentifier});
 
-        request = new Request(raw, {protocol: 'cli'});
+        const request = new Request(raw, {protocol: 'cli'});
 
-        kuzzle.pluginsManager.trigger(beforeEvent, request)
+        this.kuzzle.pluginsManager.trigger(beforeEvent, request)
           .then(updatedRequest => {
-            kuzzle.services.list.broker.listen(updatedRequest.id, onListenCB);
-            kuzzle.services.list.broker.send(kuzzle.config.queues.cliQueue, updatedRequest.serialize());
+            this.kuzzle.services.list.broker.listen(updatedRequest.id, action.onListenCB.bind(action));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 103 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)